### ISSUE TITLE
Can't build in the very top corner

### ISSUE THREAD

**Issue by darianhickman** [darian311/VillageMakeover#36]
![screen shot 2015-04-07 at 1 12 46 pm](https://cloud.githubusercontent.com/assets/1481299/7028994/f1642bb6-dd27-11e4-9118-2ad142345eb2.png)

Thanks for getting the grid operational again. Can't build the top spot even though it was green.

**Comment by darianhickman** [comment]
I tested this on umutvillage.  Will load up to dev and try again

**Comment by umutnacak** [comment]
Yes, because probably you are clicking on ui top bar which prevents tilemap
to capture mouse events.

**Comment by darianhickman** [comment]
Hmm can top bar be passthrough while the buttons in top bar still capture
click events?

On Tue, Apr 7, 2015 at 2:06 PM, umutnacak notifications@github.com wrote:

> Yes, because probably you are clicking on ui top bar which prevents tilemap
> to capture mouse events.
> 
> On Tue, Apr 7, 2015 at 8:15 PM, Darian Hickman notifications@github.com
> wrote:
> 
> > I tested this on umutvillage. Will load up to dev and try again
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/darian311/VillageMakeover/issues/36#issuecomment-90651245
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/darian311/VillageMakeover/issues/36#issuecomment-90681414
> .

## 

Darian Hickman |  (310) 439-8573  | http://www.linkedin.com/in/darianhickman

**Comment by umutnacak** [comment]
I will have a look at that.

**Comment by darianhickman** [comment]
Hey It's looking way better but still can't build at top possibly for new reason:  when moving a building the tiles redded out by that building don't change.  
![screen shot 2015-04-08 at 4 13 11 pm](https://cloud.githubusercontent.com/assets/1481299/7054441/715db430-de0a-11e4-8d00-0dbc93d94654.png)

**Comment by umutnacak** [comment]
No, I didn't do anything to solve the top bar issue. Redded out area don't
change because it shows moving object's initial position. When you decide
to put it back(don't change position) you can place it in that area. Or
move just one cell(like in this example) you can do that. Go ahead and try
it for the huts which are in the middle.
